csharp_dotnetcore/11.QnAMaker/QnAMaker/QnABot.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+
+namespace AspNetCore_QnA_Bot
+{
+    /// <summary>
+    /// Represents a bot that can process incoming activities.
+    /// For each interaction from the user, an instance of this class is called.
+    /// </summary>
+    public class QnABot : IBot
```

Comment:
The casing here is very inconsistent. We have:
QnABot
QnaBot (as the key value in the QnaMakerKey)
QnaMaker   <-- Suggest QnAMaker
QnaMakerKey  <-- Suggest QnAMakerKey
QnAServices
QnA_Bot (namespace name)
QnA Bot (in code comments below)

The product team seems to use "QnA" in their docs, so I believe we should do the same here.